src/test/java/io/cryostat/rules/RuleProcessorTest.java

```diff
@@ -238,7 +238,7 @@ void testTaskCancellationOnFailure() throws Exception {
                 new Rule.Builder()
                         .name("Test Rule")
                         .description("Automated unit test rule")
-                        .targetAlias("com.example.App")
+                        .matchExpression("com.example.App")
```

Comment:
Is it optional for the target alias match expression to be of the form `target.alias == 'com.example.App'`?

Reply:
Oh, I missed updating that spot. Thanks for catching it.

Reply:
No problem